graphdatascience/pipeline/training_pipeline.py

```diff
@@ -60,10 +66,21 @@ def configureSplit(self, **config: Any) -> Row:
         return self._query_runner.run_query(query, params)[0]
 
     def node_property_steps(self) -> List[Dict[str, Any]]:
-        return self._list_info()["modelInfo"]["featurePipeline"]["nodePropertySteps"]  # type: ignore
+        return self._list_info()["pipelineInfo"]["featurePipeline"]["nodePropertySteps"]  # type: ignore
 
     def split_config(self) -> Dict[str, Any]:
-        return self._list_info()["modelInfo"]["splitConfig"]  # type: ignore
+        return self._list_info()["pipelineInfo"]["splitConfig"]  # type: ignore
 
     def parameter_space(self) -> List[Dict[str, Any]]:
-        return self._list_info()["modelInfo"]["trainingParameterSpace"]  # type: ignore
+        return self._list_info()["pipelineInfo"]["trainingParameterSpace"]  # type: ignore
+
+    def _list_info(self) -> Row:
+        query = "CALL gds.beta.pipeline.list($name)"
+        params = {"name": self.name()}
+
+        info = self._query_runner.run_query(query, params)
+
+        if len(info) == 0:
```

Comment:
The GDS behavior for our catalogs (graph, model, pipeline alike) is that we return a a column for each object matching the name we've specified. It will be one or zero I guess.